drivers/ovsdriver.go

```diff
@@ -218,11 +240,43 @@ func (d *OvsDriver) DeleteNetwork(id, nwType, encap string, pktTag, extPktTag in
 	return sw.DeleteNetwork(uint16(pktTag), uint32(extPktTag), gateway, tenant)
 }
 
+//CreateBandwidthshaping will chaneg the format of the bandwidth and return the updated BW
```

Comment:
I think this function should be named ConvertBandwidthShaping or ParseBandwidthShaping. It would be helpful to add an example of the input format in the comments.